File: chap3/Initiales.js
// Ces programmes sont sous licence CeCILL-B V1.
var nom, i;
nom = ISNinputString();
ISNprint(nom.charAt(0));
for (i = 0; i <= nom.length - 2; i = i + 1) {
	if (nom.charAt(i)==" ") {
		ISNprint(nom.charAt(i + 1));
	}
}
ISNprintln();

File: chap3/Initiales.cpp
// Ces programmes sont sous licence CeCILL-B V1.
#include <iostream>
#include <string>

using namespace std;

int main()
{
   string nom;
   int i;
   getline(cin,nom);
   cout << nom[0];
   for (i = 0; i <= (int)nom.length() - 2; i = i + 1)
   {
      if (nom[i] == ' ')
      {
         cout << nom[i + 1];
      }
   }
   cout << endl;
}
